// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

import "components/feed/core/proto/v2/wire/feed_response.proto";

option optimize_for = LITE_RUNTIME;

message Response {
  enum ResponseVersion {
    UNKNOWN_RESPONSE_VERSION = 0;
    FEED_RESPONSE = 1;
  }
  optional ResponseVersion response_version = 1;
  optional FeedResponse feed_response = 1000;
}
